table-print/Sources/TPConfiguration.swift

```diff
@@ -16,6 +16,7 @@ public struct TPConfiguration {
     var horizontalLineSymbol = "-"
     var verticalLineSymbol = "|"
     var placeholder = "<nil>"
+    public var maxWidth: Int?
```

Comment:
Now I realize that actually all other variables need to be public as well if someone wants to modify them. But I'll do that separately, since it is not related to this issue.